LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -191,7 +194,45 @@ public List<Module> GenerateModuleList()
 
         public void CreateClassrooms(int classroomCount)
         {
-            throw new NotImplementedException();
+            foreach(Module module in moduleDictionary.Values)
```

Comment:
Why are you looping through the dictionary? it looks like you are creating a classroom for each existing module. The task was to generate as many classrooms as you specify by the input parameter classroomCount. But you're not using classroomCount in your logic. You have to choose a module randomly for each classroom. You have already a similar mechanism of assigning modules randomly in AssignModulesToStudents() or AssignModulesToInstructors(). The only difference here that you need just one module per classroom and you can assign the same module to more than one classroom.